cmd/server/config.go

```diff
@@ -23,8 +22,8 @@ var (
 
 type config struct {
 	Mode             string `json:"mode"`
-	RPCEndpoint      string `json:"rpc_endpoint"`
-	IndexerEndpoint  string `json:"indexer_endpoint"`
+	RPCBaseURL       string `json:"rpc_base_url"`
+	IndexerBaseURL   string `json:"indexer_base_url"`
```

Comment:
It seems to me that specifying these are really base urls avoids confusion with avalanchego indexer endpoint path and highlight the flexibility of using two different sources for data

Reply:
It makes sense to align the terminology with the node concepts. You should update README.md which makes references to `rpc_endpoint` configuration name, and when it is landed, it may be a good idea to mention it in the release notes.

Reply:
Done for readme